src/external/modules/returns/lib/upload-helpers.js

```diff
@@ -21,6 +21,7 @@ const getErrorMessage = key => {
     'invalid-xml': 'The selected file must use the template',
     'invalid-csv': 'The selected file must use the template',
     'invalid-type': 'The selected file must be a CSV or XML file',
+    'no-file': 'Please select a CSV or XML file',
```

Comment:
Looking at the design system (https://design-system.service.gov.uk/components/file-upload/) I wonder if the message here should be:
"Select a CSV or XML file"